content/teknologi/altinnstudio/altinn-api/_index.md

```diff
@@ -95,20 +95,28 @@ Avaliable query paramters include:
 - created (datetime)
 - visibleAfter (datetime)
 - dueBefore (datetime)
+- excludeConfirmedBy (string)
```

Comment:
would it be helpful with an example to show that the string should be org shortname?

Reply:
Yeah, maybe. Updated the parameter list a bit.